packages/backend/src/controllers/connectors/handlers.ts

```diff
@@ -43,17 +49,44 @@ export function connectorsGetRequest(req: Request, res: Response, next: NextFunc
   }
 }
 
+/**
+ * POST /api/connectors
+ *
+ * Adds a new connector with connectorService
+ * @param req Request
+ * @param res Response
+ * @param next NextFunction
```

Comment:
```suggestion
 * @param {Request} req Request
 * @param {Response} res Response
 * @param {NextFunction} next NextFunction
```

Just for consistency it would be nice to add the type definition (or remove it in all places, since it actually is not required when using typescript).

Reply:
Hm, strange :thinking: I do not know, why there are missing as they are auto generated :sweat_smile: Well spotted!